newscript.go

```diff
@@ -0,0 +1,405 @@
+	fmt.Printf("Working dir is %s\n", workingDir)
+	cmd16 := exec.Command("sudo", "apt-get", "install", "git", "golang", "build-essential", "git-core", "gitk", "git-gui", "subversion", "curl", "python2.7", "libyaml-dev", "liblzma-dev") 
+	err = cmd16.Run() 
+	if err != nil { 
+		return err
+	}
+	/*err = blankBootstick()
+	if err != nil {
+		return err
+	}*/
+	usr, err := user.Current()
+    	if err != nil {
+        	return err
+    	}
+	homeDir = usr.HomeDir
+	return nil
+
+}
+
+//User input for putting custom chrome image on bootstick
+func blankBootstick() error{
+	fmt.Printf("-------- Creating bootstick \n")
+	var imageLoc string
+	var location string 	
+	for true {
+		fmt.Printf("What image would you like to put onto your bootstick (provide location for iso file)?\n")	
+		_, err := fmt.Scanf("%s",&imageLoc)
+		if err != nil {
+			return err	
+		}
+		if _, err = os.Stat(imageLoc); err != nil{
+			fmt.Printf("Please provide a valid file name. %s has error %v", imageLoc, err)
+		} else {
+			break
+		}
+	}
+	for true {
+		fmt.Printf("Where is your bootstick (/dev/sda?)")	
+		_, err := fmt.Scanf("%s",&location)
+		if err != nil {
+			return err	
+		}
+		if _, err = os.Stat(location); err != nil{
+			fmt.Printf("Please provide a valid location name. %s has error %v", location, err)
+		} else {
+			break
+		}
+	}
+	cmd20 := exec.Command("sudo", "dd", fmt.Sprintf("if=%s", imageLoc), fmt.Sprintf("of=%s", location))
+	err := cmd20.Run()
+	if err != nil {
+		return err
+	}
+	return nil	
+}
+
+
+func cleanup() error {	
+	filesToRemove := [...]string{linuxVersion, ".git", "NiChrome", "vboot_reference"}
+	fmt.Printf("-------- Removing problematic files %v\n", filesToRemove)
+	for _,file := range filesToRemove{
+		if _, err := os.Stat(file); err != nil {
+			continue		
+		} 
+		cmd15 := exec.Command("sudo", "rm", "-r", file)
+		err := cmd15.Run()
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+
+
+func goCompatibility() error {
+	fmt.Printf("--------Checking Go Compatibility \n")
+	cmd := exec.Command("go", "version")
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err := cmd.Run()
+	if err != nil {
+		return err
+	}
+	// The string is originally in the form: go version go1.9rc2_cl165246139 linux/amd64 where 1.9 is the go version
+	termString, err := strconv.ParseFloat(strings.Split(out.String(), " ")[2][2:5], 64)
+	if err != nil {
+		return err
+	}
+	if termString > 1.7 {
+		fmt.Println("Compatible go version")
+	} else {
+		return errors.New("Please install go v1.7 or greater.")
+	}
+	return nil
+}
+
+func goGet() error {
+	fmt.Printf("--------Getting u-root \n")
+	cmd1 := exec.Command("go", "get", "github.com/u-root/u-root")
+	err := cmd1.Run()
+	//TODO: Figure why this ^ isn't working
+	/*if err != nil {
+		return err
+	}*/
+	fmt.Printf("--------Gotu-root \n")
+	gopath := fmt.Sprintf("GOPATH=%s/go", homeDir)
+	bbpath := fmt.Sprintf("GOPATH=%s/go/src/github.com/u-root/u-root/bb/bb", homeDir)
+	cmd3 := exec.Command(gopath, bbpath)
```

Comment:
you can keep reusing the same variable. 
cmd := exec.Command the first time

cmd = exec.Command later times.